internal/idler/user_idler.go

```diff
@@ -21,7 +21,7 @@ var logger = log.WithFields(log.Fields{"component": "user-idler"})
 // JenkinsServices is an array of all the services getting idled or unidled
 // they go along the main build detection logic of jenkins and don't have
 // any specific scenarios.
-var JenkinsServices = []string{"jenkins", "content-repository"}
+var JenkinsServices = []string{"jenkins"}
```

Comment:
maybe we could clean this and subsequent loop,

Reply:
I thought of the same but then, this allows us to add the c-r back without a lot of changes to the code if  needed.